index.d.ts

```diff
@@ -1516,6 +1517,7 @@ declare namespace Shopify {
         width: number;
         height: number;
         updated_at: string;
+        alt: string | null;
```

Comment:
This is not documented, https://help.shopify.com/en/api/reference/products/product_image is it available?

Reply:
It is as far as I can see in all responses (shown in documentation) involving images, did some tests and I did get them when available and as null when not set.

Reply:
Ok, thank you.